src/commands/tag/tag.ts

```diff
@@ -0,0 +1,139 @@
+import chalk from 'chalk'
+import {Command} from 'clipanion'
+import {getCIEnv} from '../../helpers/ci'
+import {retryRequest} from '../../helpers/retry'
+import {parseTags} from '../../helpers/tags'
+import {getApiHostForSite, getRequestBuilder} from '../../helpers/utils'
+
+export class TagCommand extends Command {
+  public static usage = Command.Usage({
+    description: 'Add tags to a CI Pipeline trace pipeline or job span in Datadog.',
+    details: `
+            This command when run from a supported CI provider sends an arbitrary st of key:value
```

Comment:
did you mean string? 
```suggestion
            This command when run from a supported CI provider sends an arbitrary string of key:value
```

Reply:
I think I meant to write `set` and probably missed the `e`